modules/media/jsx/editForm.js

```diff
@@ -136,6 +137,7 @@ class MediaEditForm extends Component {
           />
           <SelectElement
             name='language'
+            id='language_id'
```

Comment:
@PapillonMcGill @driusan this still seems to be an unnecessary change (removing it does not impact functionality) but since the PR fixes the issue I won't block it over this